src/pages/community/how-to-contribute.md

```diff
@@ -1,12 +1,122 @@
-# How to Contribute
+# How to Contribute to Apache Ozone
 
-**TODO [HDDS-9869](https://issues.apache.org/jira/browse/HDDS-9869) Fill in this page.**
+Thank you for your interest in contributing to Apache Ozone! This guide will help you understand the various ways you can contribute to the project and how to get started.
 
-This may end up similar to CONTRIBUTING.md in the Ozone repo, but should document all the ways to contribute outside of code as well.
+## Quick Links
 
-- Testing and filing issues
-  - This may link to the [Report an Issue](report-an-issue) page.
-- Contributing code
-  - Both Ozone backend and Recon UI/UX front end.
-- Contributing to the website
-- Graphic design work for logos or diagrams.
+- [GitHub Repository](https://github.com/apache/ozone)
+- [CONTRIBUTING.md](https://github.com/apache/ozone/blob/master/CONTRIBUTING.md) - Detailed contribution guidelines
+- [Newbie Jira Issues](https://issues.apache.org/jira/issues/?filter=12353868) - Good first issues for new contributors
+- [GitHub Discussions](https://github.com/apache/ozone/discussions) - Engage with the community
+- [Ozone Website](https://ozone.apache.org/) - Project documentation
+- [Slack Channel](http://s.apache.org/slack-invite) - Join #ozone on ASF Slack
+
+## Ways to Contribute
+
+Apache Ozone welcomes contributions in many forms, not just code. Here are various ways you can contribute:
+
+### 1. Code Contributions
+
+#### Finding Issues to Work On
+
+- **Newbie Issues**: Start with [beginner-friendly issues](https://issues.apache.org/jira/issues/?filter=12353868) that are good entry points
+- **All Open Issues**: Browse [unassigned Jira issues](https://s.apache.org/OzoneUnassignedJiras) to find something that matches your interests and skills
+
+#### Making Code Changes
+
+1. **Set Up Development Environment**
+   - Fork and clone the repository
+   - Enable the `build-branch` GitHub Actions workflow in your fork
+
+2. **Create a Branch**
+   - Ensure a Jira issue exists for your task (e.g., HDDS-1234)
+   - Create a branch for your work: `git checkout -b HDDS-1234`
+
+3. **Develop Your Changes**
+   - Follow code style guidelines (2 spaces indentation, 120-char line length)
+   - Write tests for your changes
+   - Run relevant checks from `hadoop-ozone/dev-support/checks/` directory
+
+4. **Submit Your Contribution**
+   - Push changes to your fork
+   - Wait for the `build-branch` workflow to complete
+   - Create a pull request with a clear description
+   - Update the Jira issue status to "Patch Available"
+
+### 2. Documentation
+
+- **Website Improvements**: Contribute to the [Ozone website](https://ozone.apache.org/) - see [instructions](https://cwiki.apache.org/confluence/display/OZONE/Modifying+the+Ozone+Website)
```

Comment:
Since this page is going into the new website, let's link to CONTRIBUTING.md in this repo instead. Ideally all confluence content will be obsolete and removed when this site goes live.

Reply:
Same for other links in this section.

Reply:
ack